docs/index.rst

```diff
@@ -46,26 +32,65 @@ up to speed right away.
    quick_tour
    quick_tutorial/index
 
-* :doc:`quick_tour` goes through the major features in Pyramid, covering
-  a little about a lot.
+* :doc:`quick_tour` gives an overview of the major features in Pyramid,
+  covering a little about a lot.
 
-* :doc:`quick_tutorial/index` does the same, but in a tutorial format:
-  deeper treatment of each topic and with working code.
+* :doc:`quick_tutorial/index` is similar to the Quick Tour, but in a tutorial
+  format, with somewhat deeper treatment of each topic and with working code.
 
-* To see a minimal Pyramid web application, check out
-  :ref:`firstapp_chapter`.
+* Like learning by example? Visit the official :ref:`html_tutorials` as well as
+  the community-contributed :ref:`Pyramid tutorials
+  <tutorials:pyramid-tutorials>`, which include a :ref:`Todo List Application
+  in One File <tutorials:single-file-tutorial>`.
 
-* For help getting Pyramid set up, try
-  :ref:`installing_chapter`.
+* For help getting Pyramid set up, try :ref:`installing_chapter`.
 
-* Like learning by example? Visit the official
-  :doc:`wiki tutorial <../tutorials/wiki2/index>` as well as the
-  community-contributed
-  :ref:`Pyramid tutorials <tutorials:pyramid-tutorials>`, which include
-  a :ref:`single file tasks tutorial <tutorials:single-file-tutorial>`.
+* Need help?  See :ref:`Support and Development <support-and-development>`.
 
-* Need help?  See :ref:`Support and
-  Development <support-and-development>`.
+
+.. _html_tutorials:
+
+Tutorials
+=========
+
+Official tutorials explaining how to use :app:`Pyramid` to build various types
+of applications, and how to deploy :app:`Pyramid` applications to various
+platforms.
+
+.. toctree::
+   :maxdepth: 1
+
+   tutorials/wiki2/index.rst
+   tutorials/wiki/index.rst
+   tutorials/modwsgi/index.rst
+
+
+.. _support-and-development:
+
+Support and Development
+=======================
+
+The `Pylons Project web site <http://pylonsproject.org/>`_ is the main online
+source of :app:`Pyramid` support and development information.
+
+To report bugs, use the `issue tracker
+<https://github.com/Pylons/pyramid/issues>`_.
+
+If you've got questions that aren't answered by this documentation, contact the
+`Pylons-discuss maillist <http://groups.google.com/group/pylons-discuss>`_ or
+join the `#pyramid IRC channel <irc://irc.freenode.net/#pyramid>`_.
+
+Browse and check out tagged and trunk versions of :app:`Pyramid` via the
+`Pyramid GitHub repository <https://github.com/Pylons/pyramid/>`_. To check out
+the trunk via ``git``, use this command:
+
+.. code-block:: text
+
+  git clone git@github.com:Pylons/pyramid.git
```

Comment:
You may use `https` instead of git to clone the repo... this version only works for people who have github accounts with ssh keys uploaded.